experimental/golden_response_checker.go

```diff
@@ -220,13 +219,16 @@ func CheckGoldenJSONResponse(t *testing.T, dir string, name string, dr *backend.
 }
 
 func readGoldenJSONFile(fpath string) (string, error) {
-	raw, err := ioutil.ReadFile(fpath)
+	raw, err := os.ReadFile(fpath)
 	if err != nil {
 		return "", err
 	}
+	if len(raw) < 3 {
```

Comment:
i'm probably missing something obvious, but why wouldn't this be `if len(raw) == 0 {`?

Reply:
i changed it to `== 0` :)